### PR TITLE
#460 Fixed the documentation for Table.

### PR DESCRIPTION
**Pull request by Prad06**
[ ] Wrote test for feature
[ ] Added changes to CHANGELOG.md
[ ] Bumped version number (delete if unneeded)

**Changes proposed:**
<description of the PR>

### CODE REVIEW

File: docs/tables.rst [hunk 1]
```diff
@@ -43,6 +43,7 @@ Creation
 
     Table.__init__
     Table.empty
+    Table.from_rows
```
Comment (davidwagner) [inline docs/tables.rst:46]: .from_rows is deprecated, I don't think we should add it to the docs

**Comment by davidwagner** [comment]
Good catch, good idea!  I had one small comment but if we address that, it looks good to me.  Thank you.

File: docs/tables.rst (hunk 1 above)
Comment (Prad06) [reply docs/tables.rst:46]: I will make the necessary changes! Thank you

**Review by adnanhemani — APPROVED** [review]
LGTM - thank you!

**Pull request merged**